l2geth/core/state_transition.go

```diff
@@ -243,7 +263,18 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 		}
 	}
 	st.refundGas()
-	st.state.AddBalance(evm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
+	if rcfg.UsingOVM {
+		fee, err := fees.CalculateMsgFee(st.msg, st.state, new(big.Int).SetUint64(st.gasUsed()))
+		if err != nil {
+			return nil, 0, false, err
+		}
+		if msg.CheckNonce() {
```

Comment:
What is this line for?

Reply:
This is so it doesn't log during `eth_call`